### ISSUE TITLE
Adaptive Pollution Ageing

### ISSUE THREAD

**Issue by flbuddymooreiv** [explosivegaming/scenario#101]
**Feature overview**
Name: Adaptive Pollution Ageing
TL;DR: Scale `game.map_settings.pollution.ageing` from 1 to 10 based on game progress to alleviate the pathfinder and active entity counts for the server and all clients

**Details**
Bigger base = more pollution. More pollution = more pollution-biter interface. More angry biters = more active entities. At some point, you have obviously got enough resources and defenses to defend against any attack. So spawners do help provide a challenge during expansion, but the retaliation is nothing more than a UPS drain as you're waiting for unit groups to attack as the pathfinder can only handle so much routing. Thousands of groups could take a day or more to all attack and inevitably be vaporized.

**Possible methods**
Linearly interpolate ageing from 1 to 10 over rocket launches 1 to 1000. I.e. 0,1 rocket launches = pollution ageing 1, 1000 launches = pollution ageing 10.

**Additional context**
This doesn't make environmental or logical sense, but by the time you've launched 1000 rockets, the next step is to "clear the pollution cloud" which plainly put can't really happen in a multiplayer server on a week long reset, purely due to pollution. It's sad.
